docs/blog/newsletter-may-2021.rst

```diff
@@ -0,0 +1,178 @@
+
+.. post:: May 04, 2021
+   :tags: newsletter
+
+####################################
+Write the Docs Newsletter – May 2021
+####################################
+
+Hello, Write the Docs people! There's tons to cover this month, so I'll get straight to it.
+
+We've just wrapped up our second virtual Portland conference - thanks so much of you to all who attended for making it special! If you missed it (or just want to experience it all over again), all the talks are now `live on YouTube <https://www.youtube.com/playlist?list=PLZAeFn6dfHpkCk5Fw5yLFGTqd1OJoaXhR>`__.
+
+There's plenty more conference goodness still to come in 2021, with `virtual Prague </conf/prague/2021>`__ coming on 3rd-5th October, and we've just announced that the `Australia and India virtual conference </conf/australia/2021/news/welcome/>`__ will be 2nd-3rd December. And the Call for Proposals for Prague `is open now </conf/prague/2021/news/cfp-open/>`__. Really looking forward to hearing your ideas - big or small, inspiring or entertaining.
+
+Finally, we're proud to announce our 2021 donation matching campaign. We want to make a difference globally in our work, and also encourage Write the Docs members to engage with local organizations in their region. So please check out the `blog post </blog/2021-matching-campaign/>`__, and if you'd like to donate to of our highlighted charities or a similar one local to you, get in touch.
+
+---------------------------
+Podcasts for documentarians
+---------------------------
+
+If you're looking for podcasts about technical writing, content strategy, knowledge management, and related topics, here's a list courtesy of the WTD community (with links for podcasts with dedicated websites). The number and variety of podcasts for documentarians is a delight!
+
+Folks noted one gap: podcasts from non-English-speaking countries or in languages other than English. We have one in this list, but if you know of others, please share them in the WTD Slack!
+
+General tech writing and interviews with documentarians:
+
+- `10-Minute Tech Comm <https://www.podbean.com/podcast-detail/36ttb-52a6a/10-Minute-Tech-Comm-Podcast>`__
+- `API the Docs <https://anchor.fm/api-the-docs-podcast>`_
+- `I'd Rather Be Writing <https://idratherbewriting.com/category-podcasts/>`_
+- `Inside Tech Comm with Zohra Mutabanna <https://www.insidetechcomm.show/>`_
+- `Instrktiv's Insane Instructions Show <https://instrktiv.com/en/blog/podcast/>`__
```

Comment:
If there's a problem with "Insane" we could remove this one. It was mentioned in a list of other podcasts (as in "here are some more podcasts") rather than as an outright recommendation.

Reply:
I think as somebody else's title it should be ok?

```
<!-- vale off -->
content to ignore
<!-- vale on -->
```

does the trick

Reply:
might have to surround the whole list

Reply:
I thought it might be a sensitivity thing, thank you.

Reply:
I've popped that in. I don't have a problem with it as somebody else's title